app/controllers/portal/client_logins_controller.rb

```diff
@@ -12,25 +12,34 @@ def new
     def create
       @form = RequestClientLoginForm.new(request_client_login_params)
       if @form.valid?
-        ClientEmailLoginRequestJob.perform_later(
-          email_address: @form.email_address,
-          locale: I18n.locale,
-          visitor_id: visitor_id
-        ) if @form.email_address.present?
-
-        ClientTextMessageLoginRequestJob.perform_later(
-          sms_phone_number: @form.sms_phone_number,
-          locale: I18n.locale,
-          visitor_id: visitor_id
-        ) if @form.sms_phone_number.present?
+        if @form.email_address.present?
+          ClientEmailLoginRequestJob.perform_later(
+            email_address: @form.email_address,
+            locale: I18n.locale,
+            visitor_id: visitor_id
+          )
+          session[:email_address] = @form.email_address
```

Comment:
We add this data to the session so we can show the client's email address/phone number on the page where we tell them to check their email/phone for the next step.

Note that we remove it immediately upon showing it.